release-1.8/release_team.md

```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...**
```

Comment:
Personally I would like to see it remain present.  I can shadow someone this release.